changelog/interface_contracts.dd

```diff
@@ -0,0 +1,25 @@
+Interface contracts retrieve arguments correctly
+
+Fixed regression $(LINK2 https://issues.dlang.org/show_bug.cgi?id=15984, 15984).
+
+As a side effect, contracts are not ABI compatible with previous releases.
+
+```
```

Comment:
It's Ddoc, not markdown. You need to use `---` (at least until -preview=markdown is used for building dlang.org)

Reply:
Once that is done I think this is good to go.

Reply:
done.